README.md

```diff
@@ -38,55 +34,46 @@ tools installed.
 
 At a minimum, you'll need:
 * Node (12.20)
-* NPM (7)
-* Ruby (2.7)
-* Bundler (> 2)
+* NPM (7.15)
```

Comment:
Maybe we should add also max versions. Me and @hswopeams updated npm to 8.* as some point and our package-lock.json became "uncompatible" (not really uncompatibile but it was being modified every run)

Reply:
Agree with this comment.

Reply:
this is now set in package.json

Reply:
OK for me.